File: module_beauty/src/main/java/com/gas/beauty/ui/MainActivity.java
package com.gas.beauty.ui;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import android.os.Bundle;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.gas.beauty.R;
import com.gas.beauty.article.ArticleFragment;
import com.gas.beauty.ui.article.classify.ClassifyFragment;
import com.lib.commonsdk.constants.RouterHub;

@Route(path = RouterHub.GANK_MAINACTIVITY)
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);
//        Fragment fragment= ClassifyFragment.newInstance("all");
        Fragment fragment= ArticleFragment.newInstance();

        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container,fragment)
                    .commitNow();

            fragment.setUserVisibleHint(true);

        }
    }
}

File: module_test/src/main/java/com/gas/test/learn/lesson001_lifecycle/MainActivity.kt
package com.gas.test.learn.lesson001_lifecycle

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.gas.test.R
import com.lib.commonsdk.extension.app.debug

class MainActivity : LifecycleActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_first_lifecycle)
        findViewById<View>(R.id.button6).setOnClickListener {
            startActivity(Intent(this, SecondActivity::class.java))
        }
        findViewById<View>(R.id.button7).setOnClickListener {
            startActivity(Intent(this, TransActivity::class.java))
        }
    }
}